packages/cardano-services/src/CacheService/keys.ts

```diff
@@ -0,0 +1,6 @@
+// Network Info namespace
```

Comment:
Maybe putting this under `/NetworkInfo` would be better. Also you can 'namespace' with an object, like `export const NetworkInfoCacheKey = {TOTAL_SUPPLY, ...}`

Reply:
Addressed [e470d15](https://github.com/input-output-hk/cardano-js-sdk/pull/276/commits/e470d156e91e9a5cbf074bf9a725d0b334256850)

Reply:
@IvayloAndonov can we move `packages/cardano-services/src/InMemoryCache/keys.ts` file to `packages/cardano-services/src/NetworkInfo/DbSyncNetworkInfoProvider/keys.ts`?